packages/cubejs-backend-shared/src/env.ts

```diff
@@ -733,6 +733,45 @@ const variables: Record<string, (...args: any) => any> = {
     ]
   ),
 
+  /**
+   * Tenant ID for the Azure based export bucket srorage.
+   */
+  dbExportBucketTenantId: ({
+    dataSource,
+  }: {
+    dataSource: string,
+  }) => (
+    process.env[
+      keyByDataSource('CUBEJS_DB_EXPORT_BUCKET_TENANT_ID', dataSource)
```

Comment:
If it's azure related keys could we please prefix it with Azure? For example `CUBEJS_DB_EXPORT_BUCKET_AZURE_TENANT_ID`.

Reply:
Thanks for your review, the Azure prefix has been added for the 3 new env variables.